databricks_sql_connector_core/src/databricks/sql/auth/authenticators.py

```diff
@@ -4,11 +4,17 @@
 from typing import Callable, Dict, List
 
 from databricks.sql.auth.oauth import OAuthManager
-from databricks.sql.auth.endpoint import get_oauth_endpoints, infer_cloud_from_host
+from databricks.sql.auth.endpoint import (
+    get_oauth_endpoints,
+    infer_cloud_from_host,
+)
 
 # Private API: this is an evolving interface and it will change in the future.
 # Please must not depend on it in your applications.
-from databricks.sql.experimental.oauth_persistence import OAuthToken, OAuthPersistence
+from databricks.sql.experimental.oauth_persistence import (
+    OAuthToken,
+    OAuthPersistence,
+)
```

Comment:
same as above. Please avoid format change in this PR.

Reply:
@jackyhu-db I tried running the black autoformatter. This is the result it is giving, is there some other way to do this ?

Reply:
It may be that after your changes some newer version of Black was installed, and it formats files slightly different. You can check this by checking out `main` to a new folder, setting up a devloop, and then comparing to what you have here